src/main/java/se/repos/indexing/standalone/CommandLine.java

```diff
@@ -55,11 +55,11 @@ public static void main(String[] args) throws InterruptedException {
 			// support hook style execution
 			if (options.getParentPath() != null) {
 				if (options.getParentUrl() == null) {
-					throw new CmdLineException(parser, "Daemon mode requires parent URL");
+					throw new CmdLineException(parser, "Daemon mode requires parent URL", null);
```

Comment:
Yes I figured we don't need Localizable, but the 'cause' is a nested exception which can be passed to the constructor which in this case since this is not a result of another exception can be set to null. Is that wrong?

Reply:
It is weird by the module to deprecate one variant with String (non-Localizable) and keep the others.

I am not sure null is equivalent. Looks like Throwable defaults to 'this', not 'null'. I don't know.

Reply:
```

    /**
     * Constructs a new exception with the specified detail message and
     * cause.  <p>Note that the detail message associated with
     * {@code cause} is <i>not</i> automatically incorporated in
     * this exception's detail message.
     *
     * @param  message the detail message (which is saved for later retrieval
     *         by the {@link #getMessage()} method).
     * @param  cause the cause (which is saved for later retrieval by the
     *         {@link #getCause()} method).  (A {@code null} value is
     *         permitted, and indicates that the cause is nonexistent or
     *         unknown.)
     * @since  1.4
     */
    public Exception(String message, Throwable cause) {
        super(message, cause);
    }
```

Reply:
```

    public CmdLineException(CmdLineParser parser, String message, Throwable cause) {
        super(message, cause);
        this.parser = parser;
    }
```

Reply:
I can revert it and suppress the warning nevertheless.

Reply:
It seems ok. I have now read the Javadoc on cause = this being uninitialized state.